recipe/meta.yaml

```diff
@@ -7,16 +7,18 @@ package:
 
 source:
   url: https://github.com/sharkdp/bat/archive/v{{ version }}.tar.gz
-  sha256: bb4e39efadfab71c0c929a92b82dac58deacfe2a4eb527d4256ac0634e042ed2
+  sha256: 1dd184ddc9e5228ba94d19afc0b8b440bfc1819fef8133fe331e2c0ec9e3f8e2
 
 build:
   number: 0
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
     - rust
-    - binutils              # [linux]
-    - {{native}}binutils    # [win]
+    - clangdev             # [not osx]
```

Comment:
Why do you need this?

Reply:
Otherwise build failure on windows / linux:
```
Compiling onig_sys v69.2.0
error: failed to compile `bat v0.12.1 (/home/conda/feedstock_root/build_artifacts/bat_1567500212755/work)`, intermediate artifacts can be found at `/home/conda/feedstock_root/build_artifacts/bat_1567500212755/work/target`

Caused by:
  failed to run custom build command for `onig_sys v69.2.0`

Caused by:
  process didn't exit successfully: `/home/conda/feedstock_root/build_artifacts/bat_1567500212755/work/target/release/build/onig_sys-806e1dad3070391c/build-script-build` (exit code: 101)
--- stdout
cargo:warning=couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
cargo:warning=set the LLVM_CONFIG_PATH environment variable to a valid `llvm-config` executable
```

Reply:
Then this should be in `host`, along with `llvmdev`

Reply:
Having all 3, rust, clangdev, {{ compiler('c') }} is bit strange.

Reply:
ok. done